flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/ResultPartition.java

```diff
@@ -350,18 +336,6 @@ public String toString() {
 	 * operation.
 	 */
 	void pin() {
```

Comment:
The above javadoc `The partition can only be released after each subpartition has been consumed once per pin
operation` might also be adjusted.